atd-vzv/src/views/map/CrashPointCard.js

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { PureComponent } from "react";
+import { Card, CardBody, CardText } from "reactstrap";
+import moment from "moment";
+
+export default class CrashPointCard extends PureComponent {
+  render() {
+    const { info } = this.props;
+
+    return (
+      <Card>
+        <CardBody>
+          <CardText>
+            Date/Time: {moment(info.crash_date).format("MM/DD/YYYY HH:MM A")}
+          </CardText>
+          <CardText>Fatalities: {info.death_cnt}</CardText>
+          <CardText>Serious Injuries: {info.sus_serious_injry_cnt}</CardText>
+          <CardText>
+            Modes Involved: {info.units_involved.split(" &").join(", ")}
+          </CardText>
+          <CardText>Crash ID: {info.crash_id}</CardText>
+        </CardBody>
```

Comment:
What would you think about this data presented in a two column table?
https://reactstrap.github.io/components/tables/

Reply:
I think that would look great! I'll test it out and see.

Reply:
Thanks for the table suggestion because this looks much cleaner. 👍This was the look that I tried to go after with rows and columns but that didn't work out so well. 😅 

<img width="368" alt="Screen Shot 2020-04-28 at 11 10 30 AM" src="https://user-images.githubusercontent.com/37249039/80511207-6ba3d500-8941-11ea-92c9-a852f072d515.png">

<img width="1089" alt="Screen Shot 2020-04-28 at 11 10 55 AM" src="https://user-images.githubusercontent.com/37249039/80511222-71011f80-8941-11ea-9ec6-c6b1ac64fcd3.png">